arch/sim/src/sim/sim_framebuffer.c

```diff
@@ -215,6 +219,19 @@ static int sim_closewindow(struct fb_vtable_s *vtable)
   return ret;
 }
 
+#ifdef CONFIG_FB_SYNC
+static int sim_waitforvsync(struct fb_vtable_s *vtable)
```

Comment:
let's implement in the fb driver?

Reply:
OK,  we remove it first and add it to fb driver in the next commit.